### DISCUSSION TITLE
Android version 14 error

### DISCUSSION THREAD

**Discussion by BrillianceOsprey** [community/community#112358] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My flutter project can't install on Samsung s24 ultra that is android version 14 and can't install on vivo android version 14 that always reports "conflit with existing package".

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @BrillianceOsprey ! We're happy you're here. The Discussions category is solely related to conversations around the GitHub product Discussions. 

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Flutter like [their docs](https://docs.flutter.dev/). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
